acceptance/bundle/integration_whl/interactive_single_user/script

```diff
@@ -0,0 +1,12 @@
+export DATA_SECURITY_MODE=SINGLE_USER
```

Comment:
Why are the artifacts updated on single-user clusters but not on user_isolation ones?

Reply:
No idea.